bazel/repository_locations.bzl

```diff
@@ -550,31 +549,30 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_url = "https://github.com/bazelbuild/rules_foreign_cc",
         version = "d54c78ab86b40770ee19f0949db9d74a831ab9f0",
         sha256 = "e7446144277c9578141821fc91c55a61df7ae01bda890902f7286f5fd2f6ae46",
+        release_date = "2020-10-26",
         strip_prefix = "rules_foreign_cc-{version}",
         urls = ["https://github.com/bazelbuild/rules_foreign_cc/archive/{version}.tar.gz"],
-        release_date = "2020-10-26",
         use_category = ["build"],
     ),
     rules_python = dict(
         project_name = "Python rules for Bazel",
         project_desc = "Bazel rules for the Python language",
         project_url = "https://github.com/bazelbuild/rules_python",
-        # TODO(htuch): revert back to a point releases when pip3_import appears.
-        version = "a0fbf98d4e3a232144df4d0d80b577c7a693b570",
-        sha256 = "76a8fd4e7eca2a3590f816958faa0d83c9b2ce9c32634c5c375bcccf161d3bb5",
-        strip_prefix = "rules_python-{version}",
-        urls = ["https://github.com/bazelbuild/rules_python/archive/{version}.tar.gz"],
-        release_date = "2020-04-09",
+        version = "0.1.0",
```

Comment:
Nit: prefer not combining major updates with general refactoring, but since this is only build rule, it's fine for this PR IMHO.

Reply:
This version bump is required for the refactor of the `pip3_import` to `pip_install` and is the only dependency change in the PR